src/java.naming/share/classes/com/sun/jndi/ldap/jndiprovider.properties

```diff
@@ -8,7 +8,7 @@ java.naming.factory.control=com.sun.jndi.ldap.ctl.ResponseControlFactory
 java.naming.factory.object=com.sun.jndi.ldap.obj.AttrsToCorba:com.sun.jndi.ldap.obj.MarshalledToObject
 
 # RemoteToAttrs: Turn RMI/JRMP and RMI/IIOP object into javaMarshalledObject or
```

Comment:
I adjusted this tab stop (with one space) so it aligned properly with the line above; I think that was the intention.